src/storage/test/RebuildIndexTest.cpp

```diff
@@ -511,6 +526,10 @@ TEST_F(RebuildIndexTest, RebuildEdgeIndexWithAppend) {
   RebuildIndexTest::env_->rebuildIndexGuard_->clear();
   writer->stop();
   sleep(1);
+  for (int i = 1; i <= 5; ++i) {
+    LOG(INFO) << "sleep for " << i << "s";
+    sleep(1);
```

Comment:
Why not just increase the sleep time

Reply:
because when run this case manually(watching the execution), it may be confused when sleep more than 1 seconds.